webapp/src/app/reducer.ts

```diff
@@ -17,8 +14,8 @@ export type Error = {
 };
 
 export type State = {
-  dogs: ILostDogWithPicture[] | any;
-  editedDog: ILostDogWithPicture | any;
+  dogs: ILostDogWithPicture[] | any; //if these are not any, the clear actions throw an error (only for dogs with pictures)
```

Comment:
Remind me about it later on messenger, i'll find a way since there is no way it cannot be replaced (even with something llike null)